views/package-lock.json

```diff
@@ -1,8 +1,29 @@
 {
   "name": "@oat-sa/tao-testqti",
   "version": "0.3.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
I don't see this change in the PRs we are back-porting

Reply:
it may be safer not to touch it :)

Reply:
Agreed, reverted.